### DISCUSSION TITLE
[iOS][PWA] Connection with MetaMask

### DISCUSSION THREAD

**Discussion by 40818419** [rainbow-me/rainbowkit#1901] [Q&A] [answered]
Hi, i'm currently building dApp and trying to connect a wallet to MetaMask from Home Screen (PWA), which is failing. (works in mobile safari though).
Is there any known issue?

**Comment by 0xmkh** [comment, marked as answer]
@40818419 This is a known issue on the WalletConnect side. MetaMask is a single chain wallet and that's why it's causing some issues. You can try to update your `projectId` in the best case possible. This doesn't always happen, but happens 15% of the time.
